### PR TITLE
Create Github Action automation of update-leaderboard

### PR DESCRIPTION
**Pull request by andriyDev**
Now update-leaderboard is fully automated!

This is just a hack together of `.github/workflows/twitter-bot.yml` and `.github/workflows/docs.yml`

### CODE REVIEW

File: .github/workflows/leaderboard.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+name: "update-leaderboard"
+on:
+  push:
+    tags-ignore:
+      - 'v*-beta.*'
+  release:
+    types: [published]
+jobs:
+  generate-docs:
```
Comment (medyagh) [inline .github/workflows/leaderboard.yml:9]: rename this to something else

File: .github/workflows/leaderboard.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    runs-on: ubuntu-18.04
```
Comment (medyagh) [inline .github/workflows/leaderboard.yml:10]: use more recent

File: .github/workflows/leaderboard.yml (hunk 1 above)
Comment (andriyDev) [reply .github/workflows/leaderboard.yml]: Done.

File: .github/workflows/leaderboard.yml (hunk 2 above)
Comment (andriyDev) [reply .github/workflows/leaderboard.yml]: Done.

**Pull request merged**